Create creativecommons.md

reginankenchor opened the pull request.

reginankenchor commented:
@RichardLitt kindly take a look at this

RichardLitt commented:
Thanks for posting this.

Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source? 

I think that the contact field isn't necessary - certainly not phone numbers.

reginankenchor commented:
> Thanks for posting this.
> 
> Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source?
> 
> I think that the contact field isn't necessary - certainly not phone numbers.

Okay I will check more on this and remove the contact fields. Thank you

reginankenchor commented:
> Thanks for posting this.
> 
> Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source?
> 
> I think that the contact field isn't necessary - certainly not phone numbers.

@RichardLitt  I’ve added more context regarding their activities in open education and included the relevant link as a resource. I’ve also removed the contact section as requested.

RichardLitt commented on organizations/creativecommons.md:
```diff
@@ -0,0 +1,35 @@
+# Creative Commons Overview
+
+**Website:**  
+[Creative Commons About](https://creativecommons.org/about/)
```
why the newline?

reginankenchor replied:
> why the newline?

I have adjusted this

RichardLitt commented on organizations/creativecommons.md:
```diff
@@ -0,0 +1,35 @@
+
+
+## Overview
+Creative Commons (CC) is a global non-profit organization that provides free, legally sound tools to enable sharing, collaboration, and reuse of creative works. Through its widely used open licenses, Creative Commons facilitates open access to content while ensuring that authors can maintain control over how their work is used. Its mission is to promote knowledge-sharing and creativity across aca ...
```
Could you add a note comparing this to the OSI?

reginankenchor replied:
> Could you add a note comparing this to the OSI?

Yes I have added a note comparing them, please review @RichardLitt

RichardLitt commented on organizations/creativecommons.md:
```diff
@@ -0,0 +1,35 @@
+For general inquiries, please reach out to [info@creativecommons.org](mailto:info@creativecommons.org) 
+For press/media inquiries, contact: [communications@creativecommons.org](mailto:communications@creativecommons.org).
+
+**Personnel:** CEO Anna Tumadóttir supported by staff, Board Members, Advisory Council, and Audit Committee, along with a global community of affiliates, volunteers, and legal experts.
```
I don't think volunteers should be included here. 

What do we think about this section? Could it be improved?

I'm not sure bolding a header makes sense - shouldn't it be `###`?

reginankenchor replied:
> I don't think volunteers should be included here.
> 
> What do we think about this section? Could it be improved?
> 
> I'm not sure bolding a header makes sense - shouldn't it be `###`?

Removed

RichardLitt commented on organizations/creativecommons.md:
```diff
@@ -0,0 +1,35 @@
+
+## Funders
+Creative Commons is proudly supported by contributions from the following organizations and foundations: the William and Flora Hewlett Foundation, Arcadia Fund, Argosy Foundation, Robert Wood Johnson Foundation, MHz Foundation, Michelson 20MM Foundation, Google, Microsoft, Mozilla, Patrick J. McGovern Foundation, John D. and Catherine T. MacArthur Foundation, Chan Zuckerberg Initiative, Filecoin  ...
+Coalition, Samuel H. Kress Foundation, Siegel Family Endowment, Stewart J. Guss – Attorney at Law, Open Society Foundations, Amazon Web Services, CouponFollow, DuckDuckGo, Gabriel Levin of the Levin Firm, and Akin Gump Strauss Hauer & Feld LLP.
```
Why the line break? 

I think Funders should go last.

reginankenchor replied:
> Why the line break?
> 
> I think Funders should go last.

Adjusted the location for Funders

RichardLitt commented on organizations/creativecommons.md:
```diff
@@ -0,0 +1,35 @@
+## Members and Academic Collaborations
+Creative Commons has worked with a variety of institutions worldwide, including universities, museums, libraries, and governments. Some key academic collaborations include:
+
+- **MIT’s OpenCourseWare**, using Creative Commons licenses to make educational materials freely accessible.
```
Link to these universities in the map.

RichardLitt reviewed (commented):
Some more comments. Thank you!

RichardLitt commented:
Thank you. :) Merged!
